### PR TITLE
ddl: check EnableChangeMultiSchema sysvar when deal with multi columns

### LINKED ISSUE #29785: when tidb_enable_change_multi_schema is disable，do multi schema job succeed
## Bug Report

When `tidb_enable_change_multi_schema` is disable, can add multi columns in single ddl.

### 1. Minimal reproduce step (Required)

```sql
set global tidb_enable_change_multi_schema = false;
create table t (id int primary key, c1 int);
alter table t add column (c2 int,c3 int);
```

### 2. What did you expect to see? (Required)

```sql
ERROR 8200 (HY000): Unsupported multi schema change
```

### 3. What did you see instead (Required)

```sql
mysql> alter table t add column (c2 int,c3 int);
Query OK, 0 rows affected (2.77 sec)
```

### 4. What is your TiDB version? (Required)

master

### PR DESCRIPTION
**Pull request by Defined2014**
### What problem does this PR solve?

Issue Number: close #29785

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Comment by Defined2014** [comment]
/cc @AilinKid @zimulala @morgo @xhebox 
PTAL

**Review by AilinKid — APPROVED** [review]
LGTM

File: ddl/ddl_api.go [hunk 1]
```diff
@@ -2675,6 +2675,9 @@ func (d *ddl) AlterTable(ctx context.Context, sctx sessionctx.Context, ident ast
 		switch spec.Tp {
 		case ast.AlterTableAddColumns:
 			if len(spec.NewColumns) != 1 {
+				if !sctx.GetSessionVars().EnableChangeMultiSchema {
```
Comment (zimulala) [inline ddl/ddl_api.go:2678]: Could we put this after https://github.com/pingcap/tidb/pull/29828/files#diff-b7c54f7dad7257cbe4fa04b8d76c9bed694e2e2e7c5599f45eb2172d8d4ff8a7R2648-R2671 and extract it into a function to check if it should have an `errRunMultiSchemaChanges`? I think it's a little bit more unified.

Comment (Defined2014) [reply ddl/ddl_api.go]: Done, PTAL @zimulala

**Review by zimulala — APPROVED** [review]
LGTM

**Pull request merged**